Recommend use of RC for use with docs (#3730)
microsoft/CCF: doc/build_apps/install_bin.rst

Install
-------

> CCF releases are available on the `GitHub repository release page <https://github.com/microsoft/CCF/releases/latest>`_. It is highly recommended to run `wget` directly on a release candidate with version 2.x.x as the documents are already updated for that version.

Reviewer: ```suggestion
CCF releases are available on the `GitHub repository release page <https://github.com/microsoft/CCF/releases/latest>`_. The documentation you are currently reading describes the `main` branch and behaviour available in the 2.x release candidates.
```

> CCF releases are available on the `GitHub repository release page <https://github.com/microsoft/CCF/releases/latest>`_. It is highly recommended to run `wget` directly on a release candidate with version 2.x.x as the documents are already updated for that version.

Reviewer: ```suggestion
CCF releases are available on the `GitHub repository release page <https://github.com/microsoft/CCF/releases/latest>`_. The documentation you are currently reading describes the `main` branch and behaviour available in the 2.x release candidates.
```
